#! /bin/sh /usr/share/dpatch/dpatch-run
## confdirperms.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not mess with the permissions of cupsd.conf.

@DPATCH@
diff -urNad cups-1.4.0~svn8773~/scheduler/conf.c cups-1.4.0~svn8773/scheduler/conf.c
--- cups-1.4.0~svn8773~/scheduler/conf.c	2009-08-23 12:16:58.000000000 +0200
+++ cups-1.4.0~svn8773/scheduler/conf.c	2009-08-23 18:54:25.000000000 +0200
@@ -975,8 +975,10 @@
 			     Group, 1, 1) < 0 ||
        cupsdCheckPermissions(ServerRoot, "ssl", 0700, RunUser,
 			     Group, 1, 0) < 0 ||
+       /* Never alter permissions of central conffile
        cupsdCheckPermissions(ServerRoot, "cupsd.conf", ConfigFilePerm, RunUser,
 			     Group, 0, 0) < 0 ||
+       */
        cupsdCheckPermissions(ServerRoot, "classes.conf", 0600, RunUser,
 			     Group, 0, 0) < 0 ||
        cupsdCheckPermissions(ServerRoot, "printers.conf", 0600, RunUser,
